config/routes.rb

```diff
@@ -2,5 +2,7 @@
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
-  # root "articles#index"
+  resources :users, only: [:index, :show] do
+    resources :posts, only: [:index, :show]
+  end
```

Comment:
- [ ] Good job with routes, but please make the users index page as a root route 
![image](https://user-images.githubusercontent.com/62705947/229642554-94775d09-34a9-405b-89c6-e7d7c791b23d.png)